alfresco-solrapi-client/alfresco-solrapi-client-spi/src/main/java/eu/xenit/alfresco/solrapi/client/spi/dto/GetTextContentResponse.java

```diff
@@ -0,0 +1,87 @@
+package eu.xenit.alfresco.solrapi.client.spi.dto;
+
+import java.io.InputStream;
+import java.util.function.Supplier;
+
+public class GetTextContentResponse implements Supplier<InputStream> {
+    InputStream content;
+    SolrApiContentStatus status;
+    String transformException;
+    String transformStatusStr;
+    Long transformDuration;
+    String contentEncoding;
+
+    @Override
+    public InputStream get() {
+        return content;
+    }
+
+    // no constructor, because it depends on the framework used
+
+    public static enum SolrApiContentStatus {
+        NOT_MODIFIED, OK, NO_TRANSFORM, NO_CONTENT, UNKNOWN, TRANSFORM_FAILED, GENERAL_FAILURE;
+
+        public static SolrApiContentStatus getStatus(String statusStr) {
```

Comment:
this is never used

Is this a remnant from before eu/xenit/alfresco/solrapi/client/spring/SolrApiSpringClient.java:228 ?

Reply:
It is used, it's imported in SolrApiSpringClient.